src/concrete/ml/torch/hybrid_model.py

```diff
@@ -331,8 +331,9 @@ def remote_call(self, x: torch.Tensor) -> torch.Tensor:  # pragma:no cover
             encrypted_result = inference_query.content
             decrypted_prediction = client.deserialize_decrypt_dequantize(encrypted_result)[0]
             inferences.append(decrypted_prediction)
+
         # Concatenate results and move them back to proper device
-        return torch.Tensor(numpy.array(inferences)).to(device=base_device)
+        return torch.Tensor(inferences).to(device=base_device)
```

Comment:
What's happening here?

Reply:
so, for the above changes, mypy started to complain about this typing (idk why it started on my side but whatever). The issue was that : 
- `inferences` is typed `List[numpy.ndarray]`
- but `deserialize_decrypt_dequantize` always returns a `numpy.ndarray`
- so mypy was complaining that `inferences.append(decrypted_prediction)` has some type mismatches

the `numpy.array` removal from `numpy.array(inferences)` is a mistake indeed, although it should not break anything, I'll put it back for clarity